### DISCUSSION TITLE
[Android] Can't reopen issue having triage permission

### DISCUSSION THREAD

**Discussion by htcfreek** [community/community#128926] [Mobile]
I have triage permission on a repository. And on the web site I can reopen all issues.

But in the Android app I can't. Trying to reopen an issue that is where I am not the author fails: no permission.

For me this is in the Microsoft/PowerToys repository.

**Comment by esahmed21** [comment]
Hi @htcfreek  👋 

I have tested this in iOS also and i was able to reproduce it. I will create internal issue for our team to track it and fix it.

Sorry for the inconvenience, and Thank you for reaching out to us!
